docs/3_guides/0_submitting_proofs.md

```diff
@@ -58,43 +58,41 @@ This will create the ECDSA keystore file in `~/.aligned_keystore/keystore0`
 
 - If you have the EigenLayer CLI installed, the keystore can be generated following [these](https://docs.eigenlayer.xyz/eigenlayer/operator-guides/operator-installation#import-keys) instructions. The key will be stored into `~/.eigenlayer/operator_keys`.
 
-## 2. Fund the batcher
+## 2. Send funds to Aligned
 
-To be able to send proofs to Aligned using the Batcher, the user must fund its transactions. For this, there is a simple Batcher Payment System.
+To send proofs to Aligned using the Batcher, the user must first deposit some funds in Aligned to pay for the verification of his proofs.
```

Comment:
I recommend we are explicit try to be specific about what part of Aligned we are interacting with something like `the user must first deposit some funds in Aligned to pay` -> `the user must first deposit funds into the Aligned `BatcherPaymentService` to pay for the verification. The Batcher verifies the user has available to pay for its proof before processing its proof` contract payment contract.

Reply:
The idea of this PR was actually to remove this because the user doesn't need this information here. This information is better explained in `docs/2_architecture/components/2_payment_service_contract.md`